ansible/playbooks/roles/upgrade/defaults/main.yml

```
@@ -19,3 +19,6 @@ opendistro_for_elasticsearch:
       filename: demo2epiphany-certs-migration-root-CAs.pem
 
   upgrade_state_file_path: /etc/elasticsearch/epicli-upgrade-started.state
+
+kubernetes:
+  upgrade_flag_path: /tmp/kubernetes-upgrade-in-progress{{ ver }}.flag
```

```suggestion
  upgrade_state_file_path: /var/lib/epiphany/upgrade/state/kubernetes-{{ ver }}.uncompleted
```
Since `/tmp` is cleaned up after reboot I would use `/var/lib/epiphany/upgrade/state` instead. This location is ready to use since it's created by a task from `common` role which is run before `upgrade` playbook.

Changing name to `state_file` is for naming consistency, previously we used `lock_file` or `flag_file` but IMO `state_file` is more informative/intuitive 😉

---

Done!